packages/php-wasm/cli/src/main.ts

```diff
@@ -61,8 +61,8 @@ async function run() {
 			' '
 		)} ${process.argv[1]}`;
 		// Naively replace the PHP binary with the PHP-WASM command
-		// @TODO: Don't process the command. Lean on the shell to do it, e.g. through
-		//        a PATH or an alias.
+		// @TODO: Don't process the command. Lean on the shell to do it, e.g.
+		// through a PATH or an alias.
```

Comment:
It would be nice to see "e.g." next to the subsequent text, but this is OK and to be expected for auto-fix output.